pyriemann/embedding.py

```diff
@@ -51,18 +56,18 @@ def __init__(self, n_components=2, metric="riemann", eps=None):
 
     def _get_affinity_matrix(self, X, eps):
 
-        # make matrix with pairwise distances between points
+        # make matrix with pairwise distances between matrices
         distmatrix = pairwise_distance(X, metric=self.metric)
 
         # determine which scale for the gaussian kernel
-        if self.eps is None:
+        if eps is None:
```

Comment:
There is no need to pass `eps` as a parameter, as it is already defined as class member.

Reply:
Yes, you are right.
Function `_get_affinity_matrix` mixes the treatment of the class member `self.eps` and the argument `eps`.
The idea is to harmonize this: as I'm not a fan of overwriting the `self.eps` parameter, I prefer to use `eps` everywhere in `_get_affinity_matrix`. What do you think?

Reply:
Ok. I see your point. I think it deserves a comment. May be something like this:

"We do not want to overwrite the `self.eps` attribute, to keep consistency in the output of the `get_params` method defined by sklearn. Therefore `self.eps` is passed by copy in `_get_affinity_matrix` parameters"

Reply:
I decided to simplify the code, by removing the `eps` from the `_get_affinity_matrix` parameters.